### PR TITLE
Use logger from component library

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:**

**Description of changes:**
- Necessary changes to use logger from component library.
- Component library PR: https://github.com/aws/amazon-chime-sdk-component-library-react/pull/772

**Testing**

1. How did you test these changes?
- Deployed it and checked whether we get console logs as well as logs in AWS CloudWatch in the deployed account.
- If `LoggerProvider` is not mounted, only console logger will work as it is provided by default.

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
- apps/meeting

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/meeting/serverless/src/handlers.js [hunk 1]
```diff
@@ -293,37 +293,42 @@ async function ensureLogStream(cloudWatchClient, logStreamName) {
...
     if (body.logs.length > 0) {
       for (let i = 0; i < body.logs.length; i++) {
-        var log = body.logs[i];
-        var timestampIso = new Date(log.timestampMs).toISOString();
-        var message = `${timestampIso} [${log.sequenceNumber}] [${log.logLevel}] [mid: ${body.meetingId.toString()}] [aid: ${body.attendeeId}]: ${log.message}`;
+        const log = body.logs[i];
+        const timestampIso = new Date(log.timestampMs).toISOString();
+        let message = `${body.appName} ${timestampIso} [${log.sequenceNumber}] [${log.logLevel}]`;
+        if (body.meetingId && body.attendeeId) {
+          message = `${message} [mid: ${body.meetingId.toString()}] [aid: ${body.attendeeId}]: ${log.message}`;
```
Comment (ltrung) [inline apps/meeting/serverless/src/handlers.js:328]: nit: Can we spell out meetingId and attendeeId? More space but it is clearer. mid is a shorthand for media Id in webRTC and aid is an English word :)

File: apps/meeting/src/meetingConfig.ts [hunk 2]
```diff
@@ -1,38 +1,32 @@
...
-  postLogConfig,
-  logger,
-};
+if (!['0.0.0.0', '127.0.0.1', 'localhost'].includes(location.hostname)) {
+  const postLogger = new POSTLogger({
+    url: `${BASE_URL}logs`,
+    logLevel,
+    metadata: {
+      appName: 'ChimeComponentLibraryReactDemo',
+      timestamp: Date.now().toString(), // Add current timestamp for unique AWS CloudWatch log stream generation. This will be unique per POSTLogger creation in time.
```
Comment (ltrung) [inline apps/meeting/src/meetingConfig.ts:25]: unlikely but it is not unique if 2 ppl join at the exact same time right? Though our demo does not handle that case either so its probably fine but feel like just use uuid here is better?

Also could we use external meetingId and attendeeId so we could find the log easier?

File: apps/meeting/serverless/src/handlers.js (hunk 1 above)
Comment (devalevenkatesh) [reply apps/meeting/serverless/src/handlers.js]: Yup makes sense updated!

File: apps/meeting/src/meetingConfig.ts (hunk 2 above)
Comment (devalevenkatesh) [reply apps/meeting/src/meetingConfig.ts:25]: Will have to come back to this, but will track!

**Pull request merged**